report-ng/app/src/components/timings/session-timings/session-timings.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Testerra
+ *
+ * (C) 2023, Selina Natschke, Telekom MMS GmbH, Deutsche Telekom AG
+ *
+ * Deutsche Telekom AG and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {autoinject} from 'aurelia-framework';
+import {NavigationInstruction, RouteConfig} from "aurelia-router";
+import {AbstractViewModel} from "../../abstract-view-model";
+import * as echarts from 'echarts';
+import {EChartsOption} from 'echarts';
+import "./session-timings.scss";
+import {MethodDetails, StatisticsGenerator} from "services/statistics-generator";
+import {StatusConverter} from "../../../services/status-converter";
+import moment from "moment";
+import {MethodType} from "../../../services/report-model/framework_pb";
+
+@autoinject()
+export class SessionTimings extends AbstractViewModel {
+    private _chart: echarts.ECharts;
+    private _option: EChartsOption;
+    private _executionStatistics;
+    private _methodDetails;
+    private _baseURLData;
+    private _sessionData;
+    private _hasEnded = false;
+    private _testDuration;
+    private _bars;
+
+
+    constructor(
+        private _statusConverter: StatusConverter,
+        private _statisticsGenerator: StatisticsGenerator,
+    ) {
+        super();
+        this._executionStatistics = [];
```

Comment:
This could got to declaration.
```typescript
private _executionStatistics = [];
```

Also add typing:
```typescript
private _executionStatistics:Type[] = [];
```